docs/DafnyRef/Attributes.md

```diff
@@ -344,136 +347,271 @@ recursion was explicitly requested.
+
+### 22.2.13. `{:timeLimit N}`
+Set the time limit for verifying a given function or method.
+
+### 22.2.14. `{:timeLimitMultiplier X}`
 This attribute may be placed on a method or function declaration
 and has an integer argument. If `{:timeLimitMultiplier X}` was
 specified a `{:timelimit Y}` attributed is passed on to Boogie
 where `Y` is `X` times either the default verification time limit
 for a function or method, or times the value specified by the
 Boogie `timelimit` command-line option.
 
-### 22.1.16. trigger
-Trigger attributes are used on quantifiers and comprehensions.
-They are translated into Boogie triggers.
+### 22.2.15. `{:verify false}` {#sec-verify}
+     
+Skip verification of a function or a method altogether.
+Will not even try to verify well-formedness of postconditions and preconditions.
+We discourage to use this attribute. Prefer [`{:axiom}`](#sec-axiom),
+which performs these minimal checks while not checking that the body satisfies postconditions.
 
-### 22.1.17. typeQuantifier
-The `{:typeQuantifier}` attribute must be used on a quantifier if it
-quantifies over types.
+### 22.2.16. `{:vcs_max_cost N}` {#sec-vcs_max_cost}
+Per-method version of the command-line option `/vcsMaxCost`.
+
+The [assertion batch](#sec-assertion-batches) of a method
+will not be split unless the cost of an [assertion batch](#sec-assertion-batches) exceeds this
+number, defaults to 2000.0. In
+[keep-going mode](#sec-vcs_max_keep_going_splits), only applies to the first round.
+If [`{:vcs_split_on_every_assert}`](#sec-vcs_split_on_every_assert) is set, then this parameter is useless.
+
+### 22.2.17. `{:vcs_max_keep_going_splits N}` {#sec-vcs_max_keep_going_splits}
+
+Per-method version of the command-line option `/vcsMaxKeepGoingSplits`.
+If set to more than 1, activates the _keep going mode_ where, after the first round of splitting,
+[assertion batches](#sec-assertion-batches) that timed out are split into <n> [assertion batches](#sec-assertion-batches) and retried
+until we succeed proving them, or there is only one
+single assertion that it timeouts (in which
+case error is reported for that assertion).
+Defaults to 1.
+If [`{:vcs_split_on_every_assert}`](#sec-vcs_split_on_every_assert) is set, then this parameter is useless.
+
+### 22.2.18. `{:vcs_max_splits N}` {#sec-vcs_max_splits}
+
+Per-method version of the command-line option `/vcsMaxSplits`.
+Maximal number of [assertion batches](#sec-assertion-batches) generated for this method.
+In [keep-going mode](#sec-vcs_max_keep_going_splits), only applies to the first round.
+Defaults to 1.
+If [`{:vcs_split_on_every_assert}`](#sec-vcs_split_on_every_assert) is set, then this parameter is useless.
+
+### 22.2.19. `{:vcs_split_on_every_assert}` {#sec-vcs_split_on_every_assert}
+Per-method version of the command-line option `/vcsSplitOnEveryAssert`.
+
+In the first verification round, this option will split the original [assertion batch](#sec-assertion-batches)
```

Comment:
Again my understanding is that if you JUST use `{:vcs_split_on_every_assert}`, there is only one round of verification. Let's confirm that.